notebooks/en/enterprise_cookbook_overview.md

```diff
@@ -11,7 +11,7 @@ With JupyterLab Spaces you can spin up your personal Jupyter Notebook like in Go
 For more details, read also the [JupyterLab Spaces](https://huggingface.co/docs/hub/spaces-sdks-docker-jupyter) and the [Dev Mode](https://huggingface.co/dev-mode-explorers) documentation.
 
 
-## Inference API (Serverless)  (coming soon)
+## Inference API (Serverless)
```

Comment:
can you add a hyperlink to the section page here? (I tried to do this for my Argilla notebook, like this:
`## [Data Annotation with Argilla Spaces](enterprise_cookbook_argilla)`
But this actually doesn't seem to work)

Reply:
I think since it's a section header with its own anchor tag/link, you can't add another link like you would with regular text.